nlmod/mfpackages/constant_head.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Dec  3 11:58:49 2021
+
+@author: oebbe
+"""
+
+import xarray as xr
+import numpy as np
+
+from .. import util, cache
+
+@cache.cache_netcdf
+def get_chd_at_model_edge(model_ds, idomain):
```

Comment:
we should also add a method that does this based on a shape(file)?

Reply:
Yes, that would not be too hard. The function `gdf_to_bool_data_array` from the `mgrid` module can probably help with that.

Reply:
Can be done in as part of a new issue.